src/components/ModalConfirmation.js

```diff
@@ -43,8 +44,10 @@ class ModalConfirmation extends React.Component {
 
     const { height, width } = Dimensions.get('window');
 
-    // Prevent the bottom message to be covered by some android phones
-    const marginBottom = Platform.OS === 'android' ? 40 : 16;
+    // Prevents the bottom message from being covered
+    const marginBottom = Platform.OS === 'ios'
+      ? getKeyboardAvoidingViewTopDistance() + 30
```

Comment:
Two questions here:

1. Why using `getKeyboardAvoidingViewTopDistance` when this confirmation appears in the bottom of the screen?
2. Why `+ 30`?

Reply:
1. Because the bottom offset needed for the modal is dependent on the same elements as the keyboard avoiding views: the status bar and the header height. The code could be reused on both situations.

Maybe it would be better to refactor this method's name to something like `getBottomSafeInset()`?

2. That was trial and error. Probably the same method used to the previous static value of `16`. Here is how it is shown after this code, on the iPhone 13 mini simulator:
![Modal bottom offset](https://github.com/HathorNetwork/hathor-wallet-mobile/assets/1299409/f67f1a8e-32da-4476-aa02-3ee762702e32)

Reply:
Managed to find a better solution by removing the numeric inputs on the modal height, and replacing them with `100%` calculations. Done on [d610eb4](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/325/commits/d610eb4aeec81fa5d2c21ce469025598712db351).